src/ElementTestUtilities.ts

```diff
@@ -40,6 +41,7 @@ export async function testElementAppearsXTimesById(driver :WebDriver, id :string
  */
 export async function testElementAppearsXTimesByCSS(driver :WebDriver, css :string, count :number) {
     try {
+        await driver.wait(until.elementLocated(By.css(css)), 10000);
```

Comment:
please handle count==0 case like line 21 above for function testElementAppearsXTimesById